geometry_msgs/Vector3 p_start
geometry_msgs/Vector3 p_end
# roll pitch yaw angles in degree!
geometry_msgs/Vector3 rpy_start
geometry_msgs/Vector3 rpy_end
float64 duration
---
bool finished
